src/rules/guard-define-call.ts

```diff
@@ -27,7 +27,7 @@ const rule: Rule.RuleModule = {
 
   create(context): Rule.RuleListener {
     const definedCustomElements = new Set<string>();
-
+    const sourceCode = context.sourceCode ?? context.getSourceCode();
```

Comment:
```suggestion
    const source = context.sourceCode;
```

Ah, I see that here:

https://github.com/43081j/eslint-plugin-wc/pull/137/files